code/datums/ammo/xeno.dm

```diff
@@ -403,3 +403,39 @@
 
 /datum/ammo/xeno/oppressor_tail/proc/remove_tail_overlay(mob/overlayed_mob, image/tail_image)
 	overlayed_mob.overlays -= tail_image
+
+
+
+/datum/ammo/yautja/gauntlet_hook
+	name = "chain hook"
+	icon_state = "none"
+	ping = null
+	flags_ammo_behavior = AMMO_STOPPED_BY_COVER
+	damage_type = BRUTE
+
+	damage = XENO_DAMAGE_TIER_5
+	max_range = 4
+	accuracy = HIT_ACCURACY_TIER_MAX
+
+/datum/ammo/yautja/gauntlet_hook/on_bullet_generation(obj/projectile/generated_projectile, mob/bullet_generator)
+	//The projectile has no icon, so the overlay shows up in FRONT of the projectile, and the beam connects to it in the middle.
+	var/image/hook_overlay = new(icon = 'icons/effects/beam.dmi', icon_state = "chain", layer = BELOW_MOB_LAYER)
+	generated_projectile.overlays += hook_overlay
+
+/datum/ammo/yautja/gauntlet_hook/on_hit_mob(mob/target, obj/projectile/fired_proj)
+	shake_camera(target, 5, 0.1 SECONDS)
+	var/obj/effect/beam/chain_beam = fired_proj.firer.beam(target, "chain", 'icons/effects/beam.dmi', 10 SECONDS, 10)
+
+	var/image/chain_image = image('icons/effects/status_effects.dmi', "chain")
+	target.overlays += chain_image
+
+	new /datum/effects/xeno_slow(target, fired_proj.firer, ttl = 0.5 SECONDS)
+
+	target.apply_effect(0.5, ROOT)
```

Comment:
apply effect is legacy and it should be `target.Root(0.5)` (keep in mind that any time passed into effects is multiplied by 20 for.... reasons, so this would be 1 second of root, 

could be expressed as `1 SECONDS / GLOBAL_STATUS_MULTIPLIER` for .5 seconds or `2 SECONDS / GLOBAL_STATUS_MULTIPLIER` for 1 second, up to preference.